lib/index.ts

```diff
@@ -572,16 +593,17 @@ export class UsbbootScanner extends EventEmitter {
 			// cm: 0; cm4: 3
 			if (
 				(device.deviceDescriptor.iSerialNumber === 0 ||
-				device.deviceDescriptor.iSerialNumber === 3) && !forceSecondstage
-			) {
-				debug('Sending bootcode.bin', devicePortId(device));
-				this.step(device, 0);
-				await secondStageBoot(device, endpoint);
-				// The device will now detach and reattach with iSerialNumber 1.
-				// This takes approximately 1.5 seconds
-			} else {
+					device.deviceDescriptor.iSerialNumber === 3) && !forceSecondstage
+				) {
+					debug('Sending bootcode.bin', devicePortId(device));
+					this.step(device, 0);
+					await secondStageBoot(device, endpoint);
+					// The device will now detach and reattach with iSerialNumber 1.
+					// This takes approximately 1.5 seconds
+				} else {
+				const extraFolder = this.extraFolder;
```

Comment:
There are lots of unnecessary formatting changes here and further above. Please do not commit unnecessary changes, especially if they’re by accident like in this case. It’s confusing and slightly annoying to review a diff trying to see what exactly changed and realizing it’s only formatting, especially if it happens again and again. Please review your own PR once beforehand, and don’t let things slip through your attention.

Reply:
Sorry, I'm aware of it and spend a lot of energy making sure I don't commit formats changes.
(It doesn't happen anymore in etcher & etcher-sdk as I eventually fixed the linter config and I'm making sure that I run it before every commit). 

In this repo the linter is again completely broken (it runs but don't catch any problems), I'll fix it before going forward. (I should have learned to not rely on tools by now 😓)

Reply:
It's fixed.

Extra good thing, eslint wasn't running either and did catch some problems in the old code.

Reply:
Thank you, I appreciate it.

I don’t mind refactoring/gardening commits alongside the main changes in the same PR, I actually think their a good thing: you saw something (related) that could be improved and you did so and included the changes. That’s how codebases remain alive (and not merely by keeping dependencies up to date).